### ISSUE TITLE
[Question] This works on ionic-v1

### ISSUE THREAD

**Issue by flastowizka** [ReallySmallSoftware/cordova-plugin-firebase-crashlytics#31]
This works on ionic-v1 ?

**Comment by akshayajeevan** [comment]
Yes... working for me... also tried manually logging handled exception...

**Comment by flastowizka** [comment]
> Yes... working for me... also tried manually logging handled exception...

@akshayajeevan you can show do you implemented ?


I just install this plugin and add:

```
var crashlytics = FirebaseCrashlytics.initialise();
crashlytics.logException(mensagem);
```

but i cant see in Firebase Crashanalytics

**Comment by akshayajeevan** [comment]
I have implemented the same code on my exception handling block and this is visible on console after app restart. 

Have you tried the forced crash and got the log in Firebase console?
```
var crashlytics = FirebaseCrashlytics.initialise();
crashlytics.crash();
```

**Comment by flastowizka** [comment]
@akshayajeevan i tried to force with crash();
I app was closed fine, but the log cannot show in Firebase console.

In the tab Analitycs i can show my app version in StreamView.
But in the tab Crashanalytics the log cannot display.

I tried to update google-services.json, but the same result.

**Comment by ReallySmallSoftware** [comment]
I have released a new version with updated dependencies, removing the original Fabric dependencies. Please try again and see how it goes.
